charts/datadog/templates/_container-process-agent.yaml

```diff
@@ -100,3 +100,17 @@
 {{ toYaml .Values.agents.volumeMounts | indent 4 }}
 {{- end }}
 {{- end -}}
+
+{{- define "process-agent-config-file-flag" -}}
+{{- if not .Values.agents.image.doNotCheckTag -}}
+{{- $version := .Values.agents.image.tag | toString | trimSuffix "-jmx" -}}
+{{- $length := len (split "." $version ) -}}
+{{- if and (gt $length 1) (not (semverCompare "^6.36.0 || ^7.36.0" $version)) -}}
+--config
+{{- else -}}
+--cfgpath
+{{- end -}}
+{{- else -}}
+--config
+{{- end -}}
+{{- end -}}
```

Comment:
we usually put these helpers in https://github.com/DataDog/helm-charts/blob/main/charts/datadog/templates/_helpers.tpl

Reply:
Moved :+1: